lib/todo/delete_task_command.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ToDo.DeleteTaskCommand do
+
+  def delete_task(task_number, tasks) when is_integer(task_number) do
+    delete_task(Integer.to_string(task_number), tasks)
+  end
+
+  def delete_task(task_number, tasks) do
+    number_of_tasks =  tuple_size(List.to_tuple(tasks))
+    case Integer.parse(task_number) do
+      {number, ""} when number in 1..number_of_tasks -> List.delete_at(tasks, number - 1)
+      {number, _} -> tasks
+      :error -> tasks
```

Comment:
This piece of code is duplicated in the `edit_task_command` module. Apart from removing duplication, it could also be worth trying to extract some common function that does the number parsing. It feels like the `case` statement here is almost checking that the library function `Integer.parse` works correctly which is a bit wrong.